File: trunk/web/resource/js/contact/contact-item.js
//检查ContactType的提交
function checkContactItemForm(){
	var theForm = document.getElementById("contactItemForm");
	
	if(!checkContactName()){
		document.getElementById("contactItem.name").focus();
		return;
	}
	
	if(!checkContactPinyin()){
		document.getElementById("contactItem.pinyin").focus();
		return;
	}
	
	if(!checkContactType()){
		document.getElementById("contactItem.contactType.contactTypeId").focus();
		return;
	}
	
	if(!checkEnglishName()){
		document.getElementById("contactItem.englishName").focus();
		return;
	}
	
	if(!checkNickName()){
		document.getElementById("contactItem.nickName").focus();
		return;
	}
	
	if(!checkNativePlace()){
		document.getElementById("contactItem.nativePlace").focus();
		return;
	}
	
	if(!checkAddress()){
		document.getElementById("contactItem.address").focus();
		return;
	}
	
	if(!checkZip()){
		document.getElementById("contactItem.zip").focus();
		return;
	}
	
	if(!checkSchool()){
		document.getElementById("contactItem.school").focus();
		return;
	}
	
	if(!checkSchoolAddress()){
		document.getElementById("contactItem.schoolAddress").focus();
		return;
	}
	
	if(!checkCompany()){
		document.getElementById("contactItem.company").focus();
		return;
	}
	
	if(!checkCompanyAddress()){
		document.getElementById("contactItem.companyAddress").focus();
		return;
	}
	
	if(!checkMobile1()){
		document.getElementById("contactItem.mobile1").focus();
		return;
	}
	
	if(!checkMobile1City()){
		document.getElementById("contactItem.mobile1City").focus();
		return;
	}
	
	if(!checkMobile2()){
		document.getElementById("contactItem.mobile2").focus();
		return;
	}
	
	if(!checkMobile2City()){
		document.getElementById("contactItem.mobile2City").focus();
		return;
	}
	
	if(!checkFamilyPhone1()){
		document.getElementById("contactItem.familyPhone1").focus();
		return;
	}
	
	if(!checkFamilyPhone1City()){
		document.getElementById("contactItem.familyPhone1City").focus();
		return;
	}
	
	if(!checkFamilyPhone2()){
		document.getElementById("contactItem.familyPhone2").focus();
		return;
	}
	
	if(!checkFamilyPhone2City()){
		document.getElementById("contactItem.familyPhone2City").focus();
		return;
	}
	
	if(!checkWorkPhone1()){
		document.getElementById("contactItem.workPhone1").focus();
		return;
	}
	
	if(!checkWorkPhone1City()){
		document.getElementById("contactItem.workPhone1City").focus();
		return;
	}
	
	if(!checkWorkPhone2()){
		document.getElementById("contactItem.workPhone2").focus();
		return;
	}
	
	if(!checkWorkPhone2City()){
		document.getElementById("contactItem.workPhone2City").focus();
		return;
	}
	
	if(!checkEmail1()){
		document.getElementById("contactItem.email1").focus();
		return;
	}
	
	if(!checkEmail1Note()){
		document.getElementById("contactItem.email1Note").focus();
		return;
	}
	
	if(!checkEmail2()){
		document.getElementById("contactItem.email2").focus();
		return;
	}
	
	if(!checkEmail2Note()){
		document.getElementById("contactItem.email2Note").focus();
		return;
	}
	
	if(!checkQq1()){
		document.getElementById("contactItem.qq1").focus();
		return;
	}
	
	if(!checkQq1Note()){
		document.getElementById("contactItem.qq1Note").focus();
		return;
	}
	
	if(!checkQq2()){
		document.getElementById("contactItem.qq2").focus();
		return;
	}
	
	if(!checkQq2Note()){
		document.getElementById("contactItem.qq2Note").focus();
		return;
	}
	
	if(!checkMsn1()){
		document.getElementById("contactItem.msn1").focus();
		return;
	}
	
	if(!checkMsn1Note()){
		document.getElementById("contactItem.msn1Note").focus();
		return;
	}
	
	if(!checkMsn2()){
		document.getElementById("contactItem.msn2").focus();
		return;
	}
	
	if(!checkMsn2Note()){
		document.getElementById("contactItem.msn2Note").focus();
		return;
	}
	
	if(!checkYahoo1()){
		document.getElementById("contactItem.yahoo1").focus();
		return;
	}
	
	if(!checkYahoo1Note()){
		document.getElementById("contactItem.yahoo1Note").focus();
		return;
	}
	
	if(!checkYahoo2()){
		document.getElementById("contactItem.yahoo2").focus();
		return;
	}
	
	if(!checkYahoo2Note()){
		document.getElementById("contactItem.yahoo2Note").focus();
		return;
	}
	
	if(!checkAliwang1()){
		document.getElementById("contactItem.aliwang1").focus();
		return;
	}
	
	if(!checkAliwang1Note()){
		document.getElementById("contactItem.aliwang1Note").focus();
		return;
	}
	
	if(!checkAliwang2()){
		document.getElementById("contactItem.aliwang2").focus();
		return;
	}
	
	if(!checkAliwang2Note()){
		document.getElementById("contactItem.aliwang2Note").focus();
		return;
	}
	
	if(!checkWeb1()){
		document.getElementById("contactItem.web1").focus();
		return;
	}
	
	if(!checkWeb1Note()){
		document.getElementById("contactItem.web1Note").focus();
		return;
	}
	
	if(!checkWeb2()){
		document.getElementById("contactItem.web2").focus();
		return;
	}
	
	if(!checkWeb2Note()){
		document.getElementById("contactItem.web2Note").focus();
		return;
	}
	
	if(!checkOther1()){
		document.getElementById("contactItem.other1").focus();
		return;
	}
	
	if(!checkOther1Note()){
		document.getElementById("contactItem.other1Note").focus();
		return;
	}
	
	if(!checkOther2()){
		document.getElementById("contactItem.other2").focus();
		return;
	}
	
	if(!checkOther2Note()){
		document.getElementById("contactItem.other2Note").focus();
		return;
	}
	
	if(!checkNote()){
		document.getElementById("contactItem.note").focus();
		return;
	}
	
	theForm.submit();
}

function checkContactName(){
	var o = document.getElementById("contactItem.name");
	if(Trim(o.value)==""){
		alert("姓名不能为空!");
		return false;
	}
	if(!checkInputLength(Trim(o.value), 20)){
		alert("姓名的长度不能大于20!");
		return false;
	}
	return true;
}

function checkContactPinyin(){
	var o = document.getElementById("contactItem.pinyin");
	if(Trim(o.value)==""){
		alert("拼音不能为空!");
		return false;
	}
	if(!checkInputLength(Trim(o.value), 20)){
		alert("拼音的长度不能大于20!");
		return false;
	}
	return true;
}

function checkContactType(){
	var o = document.getElementById("contactItem.contactType.contactTypeId");
	if(o.value=="0"){
		alert("请选择联系类型!");
		return false;
	}
	return true;
}

function checkEnglishName(){
	var o = document.getElementById("contactItem.englishName");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("英文名的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkNickName(){
	var o = document.getElementById("contactItem.nickName");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("昵称的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkNativePlace(){
	var o = document.getElementById("contactItem.nativePlace");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("籍贯的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkAddress(){
	var o = document.getElementById("contactItem.address");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("联系地址的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkZip(){
	var o = document.getElementById("contactItem.zip");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 10)){
			alert("邮编的长度不能大于10!");
			return false;
		}
	}
	return true;
}

function checkSchool(){
	var o = document.getElementById("contactItem.school");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("就读学校的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkSchoolAddress(){
	var o = document.getElementById("contactItem.schoolAddress");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("学校地址的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkCompany(){
	var o = document.getElementById("contactItem.company");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("工作单位的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkCompanyAddress(){
	var o = document.getElementById("contactItem.companyAddress");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("单位地址的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkMobile1(){
	var o = document.getElementById("contactItem.mobile1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("手机1的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkMobile1City(){
	var o = document.getElementById("contactItem.mobile1City");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("手机1城市的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkMobile2(){
	var o = document.getElementById("contactItem.mobile2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("手机2的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkMobile2City(){
	var o = document.getElementById("contactItem.mobile2City");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("手机2城市的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkFamilyPhone1(){
	var o = document.getElementById("contactItem.familyPhone1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("家庭电话1的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkFamilyPhone1City(){
	var o = document.getElementById("contactItem.familyPhone1City");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("家庭电话1城市的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkFamilyPhone2(){
	var o = document.getElementById("contactItem.familyPhone2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("家庭电话2的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkFamilyPhone2City(){
	var o = document.getElementById("contactItem.familyPhone2City");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("家庭电话2城市的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkWorkPhone1(){
	var o = document.getElementById("contactItem.workPhone1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("工作电话1的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkWorkPhone1City(){
	var o = document.getElementById("contactItem.workPhone1City");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("工作电话1城市的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkWorkPhone2(){
	var o = document.getElementById("contactItem.workPhone2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("工作电话2的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkWorkPhone2City(){
	var o = document.getElementById("contactItem.workPhone2City");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("工作电话2城市的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkEmail1(){
	var o = document.getElementById("contactItem.email1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("Email1的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkEmail1Note(){
	var o = document.getElementById("contactItem.email1Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("Email1备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkEmail2(){
	var o = document.getElementById("contactItem.email2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("Email2的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkEmail2Note(){
	var o = document.getElementById("contactItem.email2Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("Email2备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkQq1(){
	var o = document.getElementById("contactItem.qq1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 15)){
			alert("QQ1的长度不能大于15!");
			return false;
		}
	}
	return true;
}

function checkQq1Note(){
	var o = document.getElementById("contactItem.qq1Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("QQ1备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkQq2(){
	var o = document.getElementById("contactItem.qq2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 15)){
			alert("QQ2的长度不能大于15!");
			return false;
		}
	}
	return true;
}

function checkQq2Note(){
	var o = document.getElementById("contactItem.qq2Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("QQ2备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkMsn1(){
	var o = document.getElementById("contactItem.msn1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("Msn1的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkMsn1Note(){
	var o = document.getElementById("contactItem.msn1Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("Msn1备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkMsn2(){
	var o = document.getElementById("contactItem.msn2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("Msn2的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkMsn2Note(){
	var o = document.getElementById("contactItem.msn2Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("Msn2备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkYahoo1(){
	var o = document.getElementById("contactItem.yahoo1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("雅虎通1的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkYahoo1Note(){
	var o = document.getElementById("contactItem.yahoo1Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("雅虎通1备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkYahoo2(){
	var o = document.getElementById("contactItem.yahoo2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("雅虎通2的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkYahoo2Note(){
	var o = document.getElementById("contactItem.yahoo2Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("雅虎通2备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkAliwang1(){
	var o = document.getElementById("contactItem.aliwang1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("旺旺1的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkAliwang1Note(){
	var o = document.getElementById("contactItem.aliwang1Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("旺旺1备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkAliwang2(){
	var o = document.getElementById("contactItem.aliwang2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 20)){
			alert("旺旺2的长度不能大于20!");
			return false;
		}
	}
	return true;
}

function checkAliwang2Note(){
	var o = document.getElementById("contactItem.aliwang2Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("旺旺2备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkWeb1(){
	var o = document.getElementById("contactItem.web1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("个人主页1的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkWeb1Note(){
	var o = document.getElementById("contactItem.web1Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("个人主页1备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkWeb2(){
	var o = document.getElementById("contactItem.web2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("个人主页2的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkWeb2Note(){
	var o = document.getElementById("contactItem.web2Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("个人主页2备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkOther1(){
	var o = document.getElementById("contactItem.other1");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("其他1的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkOther1Note(){
	var o = document.getElementById("contactItem.other1Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("其他1备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkOther2(){
	var o = document.getElementById("contactItem.other2");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 100)){
			alert("其他2的长度不能大于100!");
			return false;
		}
	}
	return true;
}

function checkOther2Note(){
	var o = document.getElementById("contactItem.other2Note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 30)){
			alert("其他2备注的长度不能大于30!");
			return false;
		}
	}
	return true;
}

function checkNote(){
	var o = document.getElementById("contactItem.note");
	if(Trim(o.value)!=""){
		if(!checkInputLength(Trim(o.value), 1000)){
			alert("备注的长度不能大于1000!");
			return false;
		}
	}
	return true;
}

//删除ContactType
function deleteContactItem(contactItemId){
	if (confirm("你确定要删除联系人详细吗？")){ 
		window.location="contactItemAction!logicDeleteContactItem.action?contactItemId="+contactItemId;
	}
}

File: trunk/src/com/imanager/contact/domain/ContactItem.java
package com.imanager.contact.domain;

import java.util.Date;

import com.imanager.common.Basement;

public class ContactItem extends Basement{
	
	private int contactItemId;
	private String name;
	private String pinyin;
	private String englishName;
	private String nickName;
	private String sex;
	private int age;
	private Date birthday;
	private String marriage;
	private String nativePlace;
	private String clCountry;
	private String clProvince;
	private String clCity;
	private String clSubcity;
	private String address;
	private String zip;
	private String school;
	private String schoolAddress;
	private String company;
	private String companyAddress;
	private String mobile1;
	private String mobile2;
	private String mobile1City;
	private String mobile2City;
	private String familyPhone1;
	private String familyPhone2;
	private String familyPhone1City;
	private String familyPhone2City;
	private String workPhone1;
	private String workPhone2;
	private String workPhone1City;
	private String workPhone2City;
	private String email1;
	private String email2;
	private String email1Note;
	private String email2Note;
	private String qq1;
	private String qq2;
	private String qq1Note;
	private String qq2Note;
	private String msn1;
	private String msn2;
	private String msn1Note;
	private String msn2Note;
	private String yahoo1;
	private String yahoo2;
	private String yahoo1Note;
	private String yahoo2Note;
	private String aliwang1;
	private String aliwang2;
	private String aliwang1Note;
	private String aliwang2Note;
	private String web1;
	private String web2;
	private String web1Note;
	private String web2Note;
	private String other1;
	private String other2;
	private String other1Note;
	private String other2Note;
	private String note;
	private ContactType contactType;
	
	private String area;
	
	public int getContactItemId() {
		return contactItemId;
	}
	public void setContactItemId(int contactItemId) {
		this.contactItemId = contactItemId;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getPinyin() {
		return pinyin;
	}
	public void setPinyin(String pinyin) {
		this.pinyin = pinyin;
	}
	public String getEnglishName() {
		return englishName;
	}
	public void setEnglishName(String englishName) {
		this.englishName = englishName;
	}
	public String getNickName() {
		return nickName;
	}
	public void setNickName(String nickName) {
		this.nickName = nickName;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public int getAge() {
		return age;
	}
	public void setAge(int age) {
		this.age = age;
	}
	public Date getBirthday() {
		return birthday;
	}
	public void setBirthday(Date birthday) {
		this.birthday = birthday;
	}
	public String getMarriage() {
		return marriage;
	}
	public void setMarriage(String marriage) {
		this.marriage = marriage;
	}
	public String getNativePlace() {
		return nativePlace;
	}
	public void setNativePlace(String nativePlace) {
		this.nativePlace = nativePlace;
	}
	public String getClCountry() {
		return clCountry;
	}
	public void setClCountry(String clCountry) {
		this.clCountry = clCountry;
	}
	public String getClProvince() {
		return clProvince;
	}
	public void setClProvince(String clProvince) {
		this.clProvince = clProvince;
	}
	public String getClCity() {
		return clCity;
	}
	public void setClCity(String clCity) {
		this.clCity = clCity;
	}
	public String getClSubcity() {
		return clSubcity;
	}
	public void setClSubcity(String clSubcity) {
		this.clSubcity = clSubcity;
	}
	public String getAddress() {
		return address;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	public String getZip() {
		return zip;
	}
	public void setZip(String zip) {
		this.zip = zip;
	}
	public String getSchool() {
		return school;
	}
	public void setSchool(String school) {
		this.school = school;
	}
	public String getSchoolAddress() {
		return schoolAddress;
	}
	public void setSchoolAddress(String schoolAddress) {
		this.schoolAddress = schoolAddress;
	}
	public String getCompany() {
		return company;
	}
	public void setCompany(String company) {
		this.company = company;
	}
	public String getCompanyAddress() {
		return companyAddress;
	}
	public void setCompanyAddress(String companyAddress) {
		this.companyAddress = companyAddress;
	}
	public String getMobile1() {
		return mobile1;
	}
	public void setMobile1(String mobile1) {
		this.mobile1 = mobile1;
	}
	public String getMobile2() {
		return mobile2;
	}
	public void setMobile2(String mobile2) {
		this.mobile2 = mobile2;
	}
	public String getMobile1City() {
		return mobile1City;
	}
	public void setMobile1City(String mobile1City) {
		this.mobile1City = mobile1City;
	}
	public String getMobile2City() {
		return mobile2City;
	}
	public void setMobile2City(String mobile2City) {
		this.mobile2City = mobile2City;
	}
	public String getFamilyPhone1() {
		return familyPhone1;
	}
	public void setFamilyPhone1(String familyPhone1) {
		this.familyPhone1 = familyPhone1;
	}
	public String getFamilyPhone2() {
		return familyPhone2;
	}
	public void setFamilyPhone2(String familyPhone2) {
		this.familyPhone2 = familyPhone2;
	}
	public String getFamilyPhone1City() {
		return familyPhone1City;
	}
	public void setFamilyPhone1City(String familyPhone1City) {
		this.familyPhone1City = familyPhone1City;
	}
	public String getFamilyPhone2City() {
		return familyPhone2City;
	}
	public void setFamilyPhone2City(String familyPhone2City) {
		this.familyPhone2City = familyPhone2City;
	}
	public String getWorkPhone1() {
		return workPhone1;
	}
	public void setWorkPhone1(String workPhone1) {
		this.workPhone1 = workPhone1;
	}
	public String getWorkPhone2() {
		return workPhone2;
	}
	public void setWorkPhone2(String workPhone2) {
		this.workPhone2 = workPhone2;
	}
	public String getWorkPhone1City() {
		return workPhone1City;
	}
	public void setWorkPhone1City(String workPhone1City) {
		this.workPhone1City = workPhone1City;
	}
	public String getWorkPhone2City() {
		return workPhone2City;
	}
	public void setWorkPhone2City(String workPhone2City) {
		this.workPhone2City = workPhone2City;
	}
	public String getEmail1() {
		return email1;
	}
	public void setEmail1(String email1) {
		this.email1 = email1;
	}
	public String getEmail2() {
		return email2;
	}
	public void setEmail2(String email2) {
		this.email2 = email2;
	}
	public String getEmail1Note() {
		return email1Note;
	}
	public void setEmail1Note(String email1Note) {
		this.email1Note = email1Note;
	}
	public String getEmail2Note() {
		return email2Note;
	}
	public void setEmail2Note(String email2Note) {
		this.email2Note = email2Note;
	}
	public String getQq1() {
		return qq1;
	}
	public void setQq1(String qq1) {
		this.qq1 = qq1;
	}
	public String getQq2() {
		return qq2;
	}
	public void setQq2(String qq2) {
		this.qq2 = qq2;
	}
	public String getQq1Note() {
		return qq1Note;
	}
	public void setQq1Note(String qq1Note) {
		this.qq1Note = qq1Note;
	}
	public String getQq2Note() {
		return qq2Note;
	}
	public void setQq2Note(String qq2Note) {
		this.qq2Note = qq2Note;
	}
	public String getMsn1() {
		return msn1;
	}
	public void setMsn1(String msn1) {
		this.msn1 = msn1;
	}
	public String getMsn2() {
		return msn2;
	}
	public void setMsn2(String msn2) {
		this.msn2 = msn2;
	}
	public String getMsn1Note() {
		return msn1Note;
	}
	public void setMsn1Note(String msn1Note) {
		this.msn1Note = msn1Note;
	}
	public String getMsn2Note() {
		return msn2Note;
	}
	public void setMsn2Note(String msn2Note) {
		this.msn2Note = msn2Note;
	}
	public String getYahoo1() {
		return yahoo1;
	}
	public void setYahoo1(String yahoo1) {
		this.yahoo1 = yahoo1;
	}
	public String getYahoo2() {
		return yahoo2;
	}
	public void setYahoo2(String yahoo2) {
		this.yahoo2 = yahoo2;
	}
	public String getYahoo1Note() {
		return yahoo1Note;
	}
	public void setYahoo1Note(String yahoo1Note) {
		this.yahoo1Note = yahoo1Note;
	}
	public String getYahoo2Note() {
		return yahoo2Note;
	}
	public void setYahoo2Note(String yahoo2Note) {
		this.yahoo2Note = yahoo2Note;
	}
	public String getAliwang1() {
		return aliwang1;
	}
	public void setAliwang1(String aliwang1) {
		this.aliwang1 = aliwang1;
	}
	public String getAliwang2() {
		return aliwang2;
	}
	public void setAliwang2(String aliwang2) {
		this.aliwang2 = aliwang2;
	}
	public String getAliwang1Note() {
		return aliwang1Note;
	}
	public void setAliwang1Note(String aliwang1Note) {
		this.aliwang1Note = aliwang1Note;
	}
	public String getAliwang2Note() {
		return aliwang2Note;
	}
	public void setAliwang2Note(String aliwang2Note) {
		this.aliwang2Note = aliwang2Note;
	}
	public String getWeb1() {
		return web1;
	}
	public void setWeb1(String web1) {
		this.web1 = web1;
	}
	public String getWeb2() {
		return web2;
	}
	public void setWeb2(String web2) {
		this.web2 = web2;
	}
	public String getWeb1Note() {
		return web1Note;
	}
	public void setWeb1Note(String web1Note) {
		this.web1Note = web1Note;
	}
	public String getWeb2Note() {
		return web2Note;
	}
	public void setWeb2Note(String web2Note) {
		this.web2Note = web2Note;
	}
	public String getOther1() {
		return other1;
	}
	public void setOther1(String other1) {
		this.other1 = other1;
	}
	public String getOther2() {
		return other2;
	}
	public void setOther2(String other2) {
		this.other2 = other2;
	}
	public String getOther1Note() {
		return other1Note;
	}
	public void setOther1Note(String other1Note) {
		this.other1Note = other1Note;
	}
	public String getOther2Note() {
		return other2Note;
	}
	public void setOther2Note(String other2Note) {
		this.other2Note = other2Note;
	}
	public String getNote() {
		return note;
	}
	public void setNote(String note) {
		this.note = note;
	}
	public String getArea() {
		return area;
	}
	public void setArea(String area) {
		this.area = area;
	}
	public ContactType getContactType() {
		return contactType;
	}
	public void setContactType(ContactType contactType) {
		this.contactType = contactType;
	}

}
